create_shape.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python3
+#
+#  Copyright Alpinereplay Inc., 2024. All rights reserved.
+#  Authors: Ryan Hu
+#
+"""
+Create shapes (line counters and polygons) using the Trace Vision GraphQL API.
+
+This script demonstrates how to create shapes like line counters and polygons
+for tracking and analytics purposes.
+
+Usage:
+1.  You will need a customer ID and API key to use this script. Contact us to
+    get these. We will also share the API URL.
+2.  Run the script using the command line with the appropriate arguments:
+
+    Create a line counter in image coordinates:
+        python create_shape.py \
+            --customer_id 42 \
+            --api_key "your_api_key" \
+            --api_url "api_url" \
+            --shape_type "line" \
+            --name "entry_counter" \
+            --coordinate_type "image" \
+            --camera_id 232 \
+            --object_type "person" \
+            --coordinates "[[30.7, 30.7], [50.7, 50.7]]"
+
+    Create a polygon in world coordinates:
+        python create_shape.py \
+            --customer_id 42 \
+            --api_key "your_api_key" \
+            --api_url "api_url" \
+            --shape_type "polygon" \
+            --name "parking_area" \
+            --coordinate_type "world" \
+            --facility_id 232 \
+            --object_type "car" \
```

Comment:
Please consistently use either `"person"` or `"vehicle"` for object types to avoid confusion.
```suggestion
            --object_type "vehicle" \
```